src/widget/wtracktableview.cpp

```diff
@@ -1074,6 +1087,49 @@ void WTrackTableView::contextMenuEvent(QContextMenuEvent* event) {
             }
         }
         m_pMenu->addMenu(m_pBPMMenu);
+
+        // Track color menu only appears if at least one track is selected
+        if (indices.size()) {
+            // Get color of first selected track
+            int column = trackModel->fieldIndex(LIBRARYTABLE_COLOR);
+            QModelIndex index = indices.at(0).sibling(indices.at(0).row(), column);
+            mixxx::RgbColor::optional_t trackColor = std::nullopt;
```

Comment:
Couldn't this be simplified by invoking `mixxx::RgbColor::optional(index.data())`? I didn't notice that I already added this conversion function :see_no_evil: Applies also to `otherTrackColor`.